tests/test_grid_objects.py

```diff
@@ -96,7 +96,7 @@ def simple_state_without_object() -> State:
     """ Returns a 2x2 (empty) grid with an agent without an item """
     return State(
         Grid(height=2, width=2),
-        Agent(position=(0, 0), orientation=None, obj=Floor()),
+        Agent(position=(0, 0), orientation=Orientation.N, obj=Floor()),
```

Comment:
I assume this was purely to fix mypy errors?

Reply:
Yes